app/assets/images/thredded/follow.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
These SVGs should probably be optimized with [svgo](https://github.com/svg/svgo)